tests/admin_scripts/tests.py

```diff
@@ -118,17 +119,26 @@ def run_test(self, args, settings_file=None, apps=None):
         test_environ['PYTHONPATH'] = os.pathsep.join(python_path)
         test_environ['PYTHONWARNINGS'] = ''
 
+        additional_kwargs = dict()
+        if umask and sys.hexversion >= 0x3090000:
+            # umask setting in subprocess.run() only supported from 3+
```

Comment:
`3+` should be `3.9+` if I'm reading the `skipIf` and `hexversion` correctly?

Reply:
Oops - absolutely right, fixing. Thanks for catching.